.github/workflows/verify_consumer_pacts.yaml

```diff
@@ -26,6 +26,7 @@ on:
   push:
     branches:
       - develop
+      - iv-AJ-1233-provider-contract
```

Comment:
@ichengchang to make sure I understand --

Normally, this addition would not be needed if you're making a pull request into `develop` branch: this block makes it so that Pact verification runs on any changes to such PRs.
```
on:
  pull_request:
    branches:
      - develop
 ```
 
 Here you've made a draft PR into my feature branch, so needed to make a corresponding change to get Pact verification to run on it.

Reply:
You're absolutely correct! The `iv-AJ-1233-provider-contract` branch was added here just so I can run a provider verification test on this draft.